### PR TITLE
Improve replication template.

### PR DESCRIPTION
**Pull request by php-coder**
- replace icon by iconClass with mongo icon  (sync with `mongodb-ephemeral-template.json` from origin)
- change the default database name (sync with `mongodb-ephemeral-template.json` from origin)
- don't mention default value in description
- add reference to MongoDB docs with details about keyfile

@bparees @mfojtik @mnagy @rhcarvalho  PTAL

### CODE REVIEW

File: 2.4/examples/replica/mongodb-clustered.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "name": "mongodb-replica-example",
     "annotations": {
       "description": "MongoDB Replication Example",
-      "icon": "icon-database",
+      "iconClass": "icon-mongodb",
```
Comment (mfojtik) [inline 2.4/examples/replica/mongodb-clustered.json:8]: do we have this icon? @jhadvig

File: 2.4/examples/replica/mongodb-clustered.json [hunk 2]
```diff
@@ -43,12 +43,12 @@
     },
     {
       "name": "MONGODB_SERVICE_NAME",
-      "description": "The name of the MongoDB Service (used to DNS lookup, default: 'mongodb')",
```
Comment (mfojtik) [inline 2.4/examples/replica/mongodb-clustered.json]: we want to show this in the console (iow. what we will create when user don't change it).

File: 2.4/examples/replica/mongodb-clustered.json (hunk 1 above)
Comment (php-coder) [inline 2.4/examples/replica/mongodb-clustered.json:8]: @mfojtik Yes, we have: https://github.com/openshift/openshift-docs/issues/1329

**Comment by mfojtik** [comment]
LGTM

**Pull request merged**